common-content/en/module/logic/demo/index.md

```diff
@@ -0,0 +1,52 @@
++++
+title="Docendo Discimus"
```

Comment:
I'm not sure about the title - convince me it's worth the confusion? :)

Reply:
Oh yeah I forgot I translated it in the intro.